src/main/scripts/webpack.config.js

```diff
@@ -75,8 +75,7 @@ module.exports = {
     ),
     new webpack.ProvidePlugin({
       $: 'jquery',
-      jQuery: 'jquery',
-      BootstrapDialog: 'BootstrapDialog'
+      jQuery: 'jquery'
```

Comment:
I just wonder why we need jQuery in our code. Do not the latest EcmaScript methods provide most if not all functionality  of this library?

Reply:
Many libraries that don't have Angular binding use it.. so it might come in handy.
AFAIK it's not used anywhere in our code right now. (besides the `See More`, for smooth scroll to the bottom of `<div>`, which will be reworked either way)